learning-outcomes.md

```diff
@@ -0,0 +1,59 @@
+# Week 1 Learning Outcomes
+
+## Teamwork & Toolkit
+
+### Pair programming
+
+- [ ] **We can use pair programming techniques to work together**
+- [ ] We can work on one computer as a pair
+- [ ] We can use four different paired programming techniques
+- [ ] We can distribute commits evenly among team members
+
+---
+
+### Version control
+
+- [ ] **We can use Git and GitHub as version control for our code**
+- [ ] We can initialise a Git repository so that we can track changes to our code over time
+- [ ] We can add / stage files so that we can include these updates in our next commit
+- [ ] We can write descriptive commit messages so that our commit history tells a story
+- [ ] We can amend commit messages so that we can correct typos/mistakes before we push to GitHub
+- [ ] We can co-author commit messages so that our commit history reflects everyone’s contributions
```

Comment:
is co-authoring necessary? We may be putting too much into their first week using Git collaboratively. I'd say this could be brought in in a later week

Reply:
It probably isn't necessary in week 1, I put it here because it's something that FAC17 learned in their final week that they wish they'd learned earlier. It probably is a bit much for week 1 though. I think we can put this one in the toolkit set of LOs for a subsequent week